### DISCUSSION TITLE
Code blocks in some ipynb files not rendering well (recent regression)

### DISCUSSION THREAD

**Discussion by corrieann** [community/community#106848] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

There seems to have been a recent regression in rendering ipynb files in the GitHub file viewer.

Previously, Colab notebook exports to GitHub rendered well.  
Today, they show unstyled code blocks like so: https://github.com/googlecolab/colabtools/issues/4365

We've found a workaround, which is to include in the main metadata block the line

 `  "pygments_lexer": "ipython3"`

[https://github.com/corrieann/colabtools/commit/730f3960349883c4ad69447e91cf108edec7513e](https://www.google.com/url?q=https://github.com/corrieann/colabtools/commit/730f3960349883c4ad69447e91cf108edec7513e&sa=D&source=buganizer&usg=AOvVaw1DU_fDHB4x2wv5kNNYBGhi)

[https://github.com/corrieann/colabtools/blob/0d10fb403da33691c71d88b64504476feb2a3777/notebooks/Classify_an_image_using_Gemini.ipynb](https://www.google.com/url?q=https://github.com/corrieann/colabtools/blob/0d10fb403da33691c71d88b64504476feb2a3777/notebooks/Classify_an_image_using_Gemini.ipynb&sa=D&source=buganizer&usg=AOvVaw02rTEj22awkjet1meTz21A)

versus

[https://github.com/corrieann/colabtools/blob/730f3960349883c4ad69447e91cf108edec7513e/notebooks/Classify_an_image_using_Gemini.ipynb](https://www.google.com/url?q=https://github.com/corrieann/colabtools/blob/730f3960349883c4ad69447e91cf108edec7513e/notebooks/Classify_an_image_using_Gemini.ipynb&sa=D&source=buganizer&usg=AOvVaw3NN_IJOyny9gz2XwZVVuH1)

However, it would be nice for this to work like it used to.  :)

**Comment by gwincr11** [comment]
Ok this should be fixed.
